### ISSUE TITLE
Add_state, add_transition correspond only to the graph

### ISSUE THREAD

**Issue by kpotomkin** [blegat/HybridSystems.jl#18]
Currently there are methods: `add_state`, `add_transition`, which are correspond only to the graph of the automata, however we can end up with the graph which will have different number of locations and transitions in comparison with `modes` and `resetmaps` array.

Proposition: add methods for the `HybridSystem`, for adding and removing states(modes) and transitions(resetmaps)

**Comment by blegat** [comment]
> add methods for the HybridSystem, for adding and removing states(modes) and transitions(resetmaps)

I agree, these method should also include the reset maps, ... for the added element
